packages/react-dev-utils/launchEditor.js

```diff
@@ -92,21 +103,41 @@ function guessEditor() {
     return shellQuote.parse(process.env.REACT_EDITOR);
   }
 
-  // Using `ps x` on OSX we can find out which editor is currently running.
-  // Potentially we could use similar technique for Windows and Linux
-  if (process.platform === 'darwin') {
-    try {
-      var output = child_process.execSync('ps x').toString();
-      var processNames = Object.keys(COMMON_EDITORS);
-      for (var i = 0; i < processNames.length; i++) {
-        var processName = processNames[i];
+  // Using `ps x` on OSX or `Get-Process` on Windows we can find out which editor is currently running.
+  // Potentially we could use similar technique for Linux
+  try {
+    if (process.platform === 'darwin') {
+      const output = child_process.execSync('ps x').toString();
+      const processNames = Object.keys(COMMON_EDITORS_OSX);
+      for (let i = 0; i < processNames.length; i++) {
+        const processName = processNames[i];
         if (output.indexOf(processName) !== -1) {
-          return [COMMON_EDITORS[processName]];
+          return [COMMON_EDITORS_OSX[processName]];
+        }
+      }
+    } else if (process.platform === 'win32') {
+      const output = child_process
+        .execSync('powershell -Command "Get-Process | Select-Object Path"', {
```

Comment:
It's really annoying that Node.js doesn't have built-in abstractions for this. Hopefully better FFI support should make this more reliable at least, instead of having to parse strings.